AnnotatoSharedLibrary/Sources/AnnotatoSharedLibrary/Models/AnnotationPart.swift

```diff
@@ -0,0 +1,15 @@
+import Foundation
+
+public class AnnotationPart: Codable {
+    public var id: UUID?
+    public var height: Double
+    public var content: String
```

Comment:
Will an `AnnotationPart`'s content always be a string?

Reply:
I think nope
But is there an encompassing data type that we can use? I'm not too sure
Need to check with the rest though @sivayogasubramanian @zognin